openapi-cli/src/main/resources/cli-help/ballerina-openapi-add.help

```diff
@@ -0,0 +1,72 @@
+NAME
+       bal openapi add - Update the Ballerina.toml file with the OpenAPI tool configuration details for generating a Ballerina service or client.
+
+SYNOPSIS
+       bal openapi add [-i | --input] <openapi-contract-file-path>
+                       [-p | --package] <package-location>
+                       [--id] <service/client id>
+                       [--module] <module-name-for-generated-service/client>
+                       [--mode] <mode-type>
+                       [--tags] <tag-names>
+                       [--operations] <operation-names>
+                       [-n | --nullable]
+                       [--license] <license-file-path>
+                       [--client-methods] <resource|remote>
+
+DESCRIPTION
+       Update the Ballerina.toml file with the OpenAPI tool configuration details for generating a Ballerina source
+       (either a mock service or a client stub) from the given OpenAPI definition file and other command options.
+
+OPTIONS
+       -i, --input <openapi-contract-file-path>
+            This is a mandatory input. The given OpenAPI contract input will
+            generate a Ballerina service and the client stub files for the given
+            OpenAPI contract. The OpenAPI contract can be either a YAML or a JSON
+
+       --id <service/client id>
+            This is a mandatory input. The given service/client id will be used
+            to identify the generated service/client in the Ballerina.toml file.
+            This id will be used to generate the Ballerina service/client
+            sources cache in the target.
+
+       --module <module-name-for-generated-service/client>
+            This is a mandatory input. The given module name will be used to
+            generate the Ballerina service/client sources in the given module
+            directory.
+
+       -p, --package <package-location>
+            This is a optional input. The given package location will be used
+            to update the Ballerina.toml in the given
+            package directory.
+
+       --mode <mode-type>
+            Mode type can be 'service' or 'client'. The Ballerina service and
```

Comment:
Currently we are only supporting `client` generation right? Are we allowing to configure both using the `bal openapi add` command and return an error for `bal build` command?

Reply:
Yeah, as discussed during the design, we decided not to do any validation in command adding phase for the given value because it will validate during the bal build phase.

Reply:
Ack, so we default this option to `client` right? If so can we mention that in the description

Reply:
Yeah default client, I will add it here